src/popup/infoPopupModule.js

```diff
@@ -131,6 +131,15 @@ function getTumblrShareLink(sourceLink, imageLink) {
 }
 
 function getPopupCreatorChildNode(creatorUrl, creator) {
+  // This following code fixes the issue #106
+  // if the creatorURL doesn't exist, we do not create
+  // a anchor tag, instead we just create a paragraph tag
+  if (creatorUrl === '#') {
+    const paragraph = document.createElement('p');
+    paragraph.target = '__blank';
```

Comment:
This line does not do anything. Paragraph element does not have any attribute `target`. Please remove this.

Reply:
Oh yeah, sorry about that.